setup/management.sh

```diff
@@ -61,6 +61,41 @@ if [ ! -f $STORAGE_ROOT/backup/secret_key.txt ]; then
 	$(umask 077; openssl rand -base64 2048 > $STORAGE_ROOT/backup/secret_key.txt)
 fi
 
+
+# Download jQuery and Bootstrap local files
+echo "Installing Mail-in-a-Box system management web assets..."
```

Comment:
No need for second echo in this script. The less output the less confusing it is for users.